### PR TITLE
Data input

### PR DESCRIPTION
**Pull request by meghanhade**
Adds basic field validation to required fields, which prevent the user from progressing with missing fields. Also adds a submission data model.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,6 +35,7 @@
     "ember-load-initializers": "^1.0.0",
     "ember-resolver": "^4.0.0",
     "ember-source": "~2.14.1",
+    "ember-truth-helpers": "^1.3.0",
```
Comment (hanbyul-here) [inline package.json]: I don't see any of this imported. Do we need `ember-truth-helpers`?

**Comment by hanbyul-here** [comment]
- Can we make the landing page (`localhost:4200`) to display the actual first page (or at least redirecting?) ? I am not sure how to navigate from this page.

**Comment by meghanhade** [comment]
@hanbyul-here I had it set up to do that, but it was redirecting to the beginning of the flow on every reload, and Ember auto-reloads with every save. I can poke at it to see if there's a way to avoid this, though.

In the meantime, you can go to:
http://localhost:4200/#/intro

File: package.json (hunk 1 above)
Comment (meghanhade) [reply package.json]: Yes, I prefer to use ember-truth-helpers. It makes conditional templating much easier.

**Pull request merged**